sandbox-web/src/main/java/com/exadel/sandbox/team5/EmployeeRestController.java

```diff
@@ -0,0 +1,39 @@
+package com.exadel.sandbox.team5;
+
+import com.exadel.sandbox.team5.service.EmployeeService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Set;
+
+@RestController
+@RequestMapping("/employee")
+@RequiredArgsConstructor
+public class EmployeeRestController {
+    private final EmployeeService employeeService;
+
+    @PutMapping("/{id}/Favorites/add")
+    public String addFavorites(@PathVariable Long id, @RequestBody Set<Long> discountIds) {
+        employeeService.addFavorites(id, discountIds);
+        return "Favorites was updated";
```

Comment:
Just boolean or object if we have it.  For UI it will be more easy to check boolean than write code like if ```( "Favorites was updated" === resp.body) ```

Reply:
okay